straxen/matplotlib_utils.py

```diff
@@ -51,16 +61,18 @@ def plot_pmts(
             show_tpc=show_tpc,
             vmin=vmin, vmax=vmax,
             **kwargs)
+        if (array_i == 0) and show_axis_labels:
+            ax.set_xlabel('x [cm]')
+            ax.xaxis.set_label_coords(1.035, -0.075)
```

Comment:
I have chosen the coordinates such that visually the x label is between the top and bottom plots. If you have better idea please go ahead

Reply:
Ciao Daniel, I have tried to look into transform, but I didn't figure out how to implement here